activejob/lib/active_job/core.rb

```diff
@@ -43,6 +43,16 @@ module Core
     # Track when a job was enqueued
     attr_accessor :enqueued_at
 
+    # Track whether the adapter received the job successfully.
+    attr_writer :successfully_enqueued
```

Comment:
This should not be used by user so let's add a `# :nodoc:` on its side.
```suggestion
    attr_writer :successfully_enqueued # :nodoc:
```